docs/usage/configuration-options.md

```diff
@@ -1683,10 +1683,13 @@ Possible values and meanings:
 
 - `auto`: Renovate will autodetect the best setting. Defaults to `conflicted` unless the repository has a setting requiring PRs to be up to date with the base branch
 - `never`: Renovate will never rebase the branch
-- `conflicted`: Renovate will rebase only if the branch is conflicted
+- `conflicted`: Renovate will rebase only if the branch is conflicted.
 - `behind-base-branch`: Renovate will rebase whenever the branch falls 1 or more commit behind its base branch
 
-Note: this field replaces the previous fields of `rebaseConflictedPrs` and `rebaseStalePrs`.
+`rebaseWhen=conflicted` is not recommended if you have enabled Renovate automerge, because:
+
+- It could result in a broken base branch if two updates are merged one after another without testing them combined
```

Comment:
```suggestion
- It could result in a broken base branch if two updates are merged one after another without testing them together
```

"testing them combined" is a bit of a weird phrase, maybe my suggestion it better.

The main idea we're trying to communicate here seems to be: "If you tell Renovate to rebase only when conflicted, you're not testing out if your code on `main` works with the code in the PR. So stuff might break because you've not tested if your code actually fits together. Keep in mind that non-conflicted PRs can still have logical errors between their code and the code on `main`."